src/extlibs/hrvo/hrvo_agent.h

```diff
@@ -105,6 +105,22 @@ class HRVOAgent : public Agent
      */
     void insertNeighbor(std::size_t agentNo, float &rangeSq);
 
+    /**
+     * Get a list of circles which represent the new velocity candidates
+     * @param circle_rad The radius of the circles which represent candidates
+     * @return list of circles which represent the new velocity candidates
+     */
+    std::vector<Circle> getCandidateVelocitiesAsCircles(
+        const float circle_rad = 0.03f) const;
+
+    std::vector<Polygon> getVelocityObstaclesAsPolygons() const;
+
+    /**
+     * Update preferred speed of Agent
+     * @param pref_speed New preferred speed
+     */
+    void setPrefSpeed(float pref_speed);
```

Comment:
nit: perhaps this should be called `setPreferredSpeed`

and it would be nice to expand on the comment a bit, I am not sure what the preferred speed does

Reply:
I updated the method name, and added a bit more description to the javadoc, and to the declaration of the `pref_speed` variable in `Agent.h`.